### ISSUE TITLE
Insufficient Color Contrast in Pagination Links

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#6777]
## Overview
Color contrast does not meet minimum requirement
AA 1.4.3 Contrast (Minimum)
AAA 1.4.6 Contrast (Enhanced)

> There is insufficient color contrast between the text and background on this page to meet the minimum contrast requirement.

The gray text for the active page in pagination lacks enough contrast to meet AA:
<img width="325" height="81" alt="Image" src="https://github.com/user-attachments/assets/2c17df16-6318-44a4-b422-581e482f4be3" />

<img width="331" height="315" alt="Image" src="https://github.com/user-attachments/assets/db9bd705-999a-42c5-9b0d-d87ca46ea3ba" />

## Done Looks Like
- [ ] Contrast for inactive/current page in the pagination section meets WCAG AA levels
